### PR TITLE
Add documentation for using systemd

### PR DESCRIPTION
**Pull request by amgarnier**
updating Documentation for using systemd instead of init.d for automatically starting device on boot

### CODE REVIEW

File: Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+`
+/home/pi/myiotappfolder/myiotapp
+`
+
+The app name is `<myiotapp>` in this documentation you would replace the entire string including the carrots in your files.
+
+Make sure to make your app ```<myiotapp>``` is executable by using the command:
+
+```shell
+# Requires root permissions
```
Comment (krwq) [inline Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md:16]: if this app runs as root you should mention that app and all its dependencies should be made writable for root only

Comment (krwq) [reply Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md:16]: you already did in the bottom, make sure to add a link just before this

**Review by krwq — APPROVED** [review]
small comment

File: Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md (hunk 1 above)
Comment (amgarnier) [reply Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md:18]: Added a link

File: Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ For the purpose of this document let's assume you have deployed your app under:
 
 The app name is `<myiotapp>` in this documentation you would replace the entire string including the carrots in your files.
 
+To run this you will need to make your app run as root please read the section on  [Security Considerations](#Security considerations)
```
Comment (krwq) [inline Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md]:
```suggestion
To run this you will need to make your app run as root please read the section on  [Security Considerations](#security-considerations)
```

Comment (krwq) [reply Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md]: the current link doesn't seem to work (you can test by clicking on 3 dots in this PR and clicking "View File")

Comment (amgarnier) [reply Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md]: that seemed to fix it thank you. just pushed a new commit

**Review by krwq — APPROVED** [review]

**Pull request merged**